src/components/pages/OSPConsent/CompanyDetails.tsx

```diff
@@ -227,6 +240,10 @@ export const CompanyDetails = ({
       [t('osp.city'), companyDetails?.city ?? ''],
       [t('osp.region'), companyDetails?.region ?? ''],
       [t('osp.country'), companyDetails?.countryAlpha2Code ?? ''],
+      ...(companyDetails?.uniqueIds?.map((id) => {
+        const strType = checkUniqueIdType(id.type as string)
+        return [strType, id.value]
+      }) ?? []),
```

Comment:
I suggest more compact:

```suggestion
      ...(companyDetails?.uniqueIds?.map((id) =>
        [checkUniqueIdType(id.type as string), id.value]
      ) ?? []),
```